monai/metrics/panoptic_quality.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+import torch
+
+from monai.metrics.utils import do_metric_reduction, remap_instance_id
+from monai.utils import MetricReduction, optional_import
+
+from .metric import CumulativeIterationMetric
+
+linear_sum_assignment, _ = optional_import("scipy.optimize", name="linear_sum_assignment")
+
+__all__ = ["PanopticQualityMetric", "compute_panoptic_quality"]
+
+
+class PanopticQualityMetric(CumulativeIterationMetric):
+    """
+    Compute panoptic quality between two tensors.
```

Comment:
Could we described briefly what the metric actually computes and what applications it would be for?